index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nakul's Recipes</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="content">
+    <div class="block-one">
+        <div class="header">
+            <p class="logo"><strong></strong></p>
+            <ul class="header-links">
+                <li><a href="#">Home</a></li>
+                <li><a href="./about.html">About</a></li>
+            </ul>
+        </div>
```

Comment:
Use the `<header>` tag to denote where the header is instead of the `<div>` you are currently using

Reply:
Also, you might want to move your navbar out of the "block-one" div so the navbar looks the same across all your pages. Right now, it's positioned differently on your homepage